### ISSUE TITLE
Login page should be using common ErrorAlert component when showing errors

### ISSUE THREAD

**Issue by junzhengca** [junzhengca/tinybox#9]
Right now the login page uses Alert component directly from Chakra UI, we should be using ErrorAlert custom component instead:

https://github.com/junzhengca/tinybox/blob/main/packages/web/src/app/pages/LoginPage.tsx#L65

https://github.com/junzhengca/tinybox/blob/main/packages/web/src/app/components/ErrorAlert.tsx

**Comment by diballesteros** [comment]
Can I take this issue?

**Comment by junzhengca** [comment]
@diballesteros absolutely! I will assign this to you, just provide updates when you have time. and if you are not interested in it anymore, please comment so I can ressign.

**Comment by diballesteros** [comment]
Sorry don't think I'll be able to tackle it you can go ahead and unassign me
